Sources/Turf/Geometries/LineString.swift

```diff
@@ -78,9 +78,63 @@ extension LineString {
     }
     
     /**
-     Returns the portion of the line string that begins at the given coordinate and extends the given distance along the line string.
+     Returns the portion of the line string that begins at the given start distance and extends the given stop distance along the line string.
      
-     This method is roughly equivalent to the [turf-line-slice-along](https://turfjs.org/docs/#lineSliceAlong) package of Turf.js ([source code](https://github.com/Turfjs/turf/tree/master/packages/turf-line-slice-along/)), except that it accepts a starting position instead of a starting distance along the line string.
+     This method is roughly equivalent to the [turf-line-slice-along](https://turfjs.org/docs/#lineSliceAlong) package of Turf.js ([source code](https://github.com/Turfjs/turf/tree/master/packages/turf-line-slice-along/)).
+     */
+    public func lineSliceAlong(startDistance: LocationDistance, stopDistance: LocationDistance) -> LineString? {
+        let coordinates = self.coordinates
```

Comment:
When porting a method from Turf.js, add a comment to the beginning that has the permalink to the Turf.js source code to make it easier to compare the ported code to the original code.